policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/benefit_base/senior.yaml

```diff
@@ -0,0 +1,22 @@
+description: Maine allows for this property tax fairness credit benefit base for senior 
+
+values:
+  2021-01-01: 0 
+  2024-01-01: 4_000
```

Comment:
This value comes effect in 2024, should I create a bool parameter for it?

Reply:
no thats ok. but turn this into a scale parameter based on age. the legal code defines it separately from the cap one so we need to reflect that.